linchpin/linchpin.conf

```diff
@@ -88,3 +88,18 @@ level = logging.INFO
 
 [ansible]
 console = True
+
+[playbook_pre_states]
+up = preup
+destroy = predestroy
+
+[playbook_post_states]
+up = postup
+destroy = postdestroy
+
```

Comment:
Since there aren't any current `substates`, couldn't the above sections be in the `[states]` section below? Something like:

```
[states]
preup = preup
postup = postup
predestroy = predestroy
postdestroy = postdestroy
```
And then just add a `[substates]` section later when needed?